tests/test_transit_coverage.py

```diff
@@ -1,17 +1,26 @@
 import numpy as np
-import pytest
 from numpy.testing import assert_allclose
-
+import pytest
 from exovetter import transit_coverage
 
+def test_coverage1():
+    time = np.arange(0, 100, step=.01)
+    p_day = 3
+    epoch = 3
+    dur_hour = 5
+
+    coverage, h, b = transit_coverage.calc_coverage(
+        time, p_day, epoch, dur_hour, ndur=2, nbins=10)
+
```

Comment:
I think this is already covered under `(0.01, 3, 3, 5, 1)` in parameterization below.
```suggestion
```

Reply:
indeed. I'll remove.